app/component/RentalVehicleContent.js

```diff
@@ -0,0 +1,178 @@
+import PropTypes from 'prop-types';
```

Comment:
Scooter page layout seems broken - e.g.  back arrow is huge and overlaps other content. I guess it should look similar to citybike page.
![image](https://github.com/HSLdevcom/digitransit-ui/assets/13776096/54179df8-9071-4dec-a4a0-165df41a6c9f)

Reply:
Yeah, not sure how this was missed but I adjusted it. There are no exact design pictures for the desktop scooters, only mobile version. I also tried to make sure it closely resembles the design pictures (there was a line missing).